header.php

```diff
@@ -46,7 +46,24 @@
         </ul>
       </nav>
     </div>
+   </div>
 
 <!--  <img src="http://Setupshop/wp-content/uploads/2016/09/SA.png" alt="sa-flags">  -->
 
-<?php dynamic_sidebar( 'Filters' ); ?>
\ No newline at end of file
+<!--<?php dynamic_sidebar( 'Filters' ); ?>-->
+
+<div class="container">
+  <section>
+  	<article id="desc">
```

Comment:
Article does not seem to be the right element to use here. What is the content of the description going to be and what is the content of main going to be?